### ISSUE TITLE
[Critical] Cannot load content after upgrade to Version 1.1.2 (6222) on Mac 12.2.1

### ISSUE THREAD

**Issue by legoming** [typora/typora-issues#5069]
**Describe the bug**

**To Reproduce**
1. install Typora update to Version 1.1.2 (6222)
2. Typora relaunched once installed
3. File content is empty (while the file is not empty in fact)

**Screenshots / Screencasts**
File content show OK on "Source Mode"
<img width="1464" alt="image" src="https://user-images.githubusercontent.com/8018372/154621570-c0b5aa57-7096-48bc-a63c-b6780146d5b0.png">

Show nothing on normal mode
<img width="1405" alt="image" src="https://user-images.githubusercontent.com/8018372/154621679-90409c1c-b197-406f-9e1a-9254c00c5f1a.png">

**Desktop (please complete the following information):**
 - OS: Mac 12.2.1

**Typora Version**
Version 1.1.2 (6222)

**Comment by legoming** [comment]
<img src="/Users/lego/Desktop/logo2.svg" height="50" align="left">

<h1>000411B - Markdown Template Help</h1>

[toc]

## Output

### deleted line

~~deleted chars~~

### rm file from git

从 cache 中移除，同时把文件类型加入到 .gitignore 中。

```
git rm --cached .DS_Store aio_custom/.DS_Store
```

### Details

<details>
<summary>Test
</summary>
just to test details' open close    
It is highly recommended that you avoid supplying related data as multiple separate <meta-data> entries. Instead, if you have complex data to associate with a component, store it as a resource and use the resource attribute to inform the component of its ID.
</details>

### Section Title

<section>this is section title without index in toc 这是一个小节标题「这类标题不会被生成到 TOC」</section>

### Highlight & Links

<hl>I'm Highlight</hl> 高亮，一般用于文本中某一段文字的高亮

<hlred>I'm Red</hlred> <hlred>红色3D高亮</hlred>

<hlgreen>I'm Green</hlgreen> <hlgreen>绿色3D高亮</hlgreen>

<hlblue>I'm Blue</hlblue> <hlblue>蓝色3D高亮</hlblue>

<hlgray>I'm Gray</hlgray> 灰色3D高亮 <hlgray>灰色高亮</hlgray>

<hllink>[Google Me](https://www.google.com)</hllink> 链接

[Google Me](https://www.google.com) 这个是一个常规链接

### Prompt

<note>This type of callout is an ordinary note or tip.
<br>这种类型的标注是一个普通的注释或提示。
</note>

<important>This type of callout is an important indicator.</important>

<caution>This type of callout suggests proceeding with caution.<br>这种类型的标注建议谨慎行事。</caution>

<warning>This type of callout is stronger than a Caution; it means "Don't do this."<br>这种类型的标注比警示强;意思是 "不要这样做"。</warning>

<fixme>This type of callout is an issue waiting for fix.<br>这种类型的标注说明这里有一个已知问题，需要解决。</fixme>

<success>This type of callout describes a successful action or an error-free status. Used only in interactive or dynamic content; don't use in ordinary static pages.
<br>这种类型的标注描述了成功的操作或无错误的状态。仅用于交互式或动态内容;不要在普通静态页面中使用。
</success>

<point>This type of callout defines an important concept.
<br>这种类型的标注定义了一个重要的概念。
</point>

<term>This type of callout defines important terminology.
<br>这种类型的标注定义了重要的术语。
</term>

<objective>This type of callout defines the goal of a procedure.
<br>这种类型的标注定义了一个目标。
</objective>

<dogfood>This type of callout is for notes that apply temporarily during internal dogfood testing. Remove all Dogfood callouts before making a document publicly visible.</dogfood>

## How to Write

### Section Title

<point>\<section>\</section>

wrtie in this way:

> \<section>this is section title without index in toc\</section>

will get output:

<section>this is section title without index in toc</section>

### Highlight & Links

#### Highlight

wrtie in this way:

> \<hl>Highlighted\</hl>

will get output:

<hl>Highlighted</hl>

replace \<<font color=dark>*hl*</font>> with **hlred** | **hlgreen** | **hlblue** | **hlgray**

#### Links

write in this way:

> \<hllink>\[Google Me\](https://www.google.com) \</hllink>

will get output:

<hllink>[Google Me](https://www.google.com)</hllink>

### Prompt

wrtie in this way:

> \<note>This type of callout is an ordinary note or tip.

or 

> \<note>This type of callout is an ordinary note or tip.\</note>

will get output:

<note>This type of callout is an ordinary note or tip.</note>


replace \<<font color=dark>*note*</font>> with **caution** | **warning** | **success** | **point** | **term** | **objective** | **dogfood**

### Resize Image

> \<img src="/Users/ming/workspace/docimage/rom_simlock.png" alt="drawing" style="width:300px;"/>

### Mark

<mark style="background:green;color:white">  ROM5.0 行为变更 </mark>

<mark style="background:blue;color:white">  ROM4.0 行为变更 </mark>

<mark style="background:grey;color:white">  ROM4.0 行为变更 </mark>

### Button

<button style="background:green;color:white;height=20px">Click Me</button>

## Table

<div class="devsite-table-wrapper">
<table>
    <tbody>
    <tr>
      <th>方法</th>
      <th>返回值</th>
      <th>参数列表</th>
      <th>供应模块</th>
      <th>供应类</th>
    </tr>

    <tr>
      <td rowspan="2"> tctSupplyIccDepersonalization </td>
      <td> void </td>
      <td> String netpin<br>String type<br>Message result<br>int phoneId </td>
      <td> tct.telephony.support </td>
      <td> com.tct.telephony.TctTelephonyManager </td>
    </tr>
    
    <tr>
      <td colspan="4"> 
            <section> 方法实现说明 </section>
            <br><strong>netpin 为空时</strong>：从 modem 获取 sim lock retry count
            <br><strong>netpin 不为空时</strong>：解锁 sim lock
            <br>
            <br><section> 参数列表说明 </section>
            <br><strong>netpin</strong>：用户输入的解锁码，或者 null
            <br><strong>type</strong>：PersoSubState
            <br><strong>result</strong>：用于接收返回值的 Message 回调
            <br><strong>phoneId</strong>：当前 sim 卡的 phone id
      </td>
    </tr>
    
    </tbody>
    
    </table>
</div>

## Graphviz

## Format

```graphviz
digraph whole {
    subgraph cluster_0 {
        subgraph cluster_sample {
            label="Sample"
            bgcolor=lightgray
            node [shape=egg]
            subgraph cluster_sample_parent {
            label=""
            parent
            }
            subgraph cluster_sample_inherit {
            label="Inherit"
            child
            }
            child -> parent [color=purple arrowhead=empty label="            " penwidth=3]
            subgraph cluster_sample_invoke {
            label="Invoke"
            classA -> parent [arrowhead=vee]
            }
            subgraph cluster_sample_rely {
            label="Rely on"
            classB -> parent [style=dashed]
            }
        }
    }
}
```

### 简单关系图

```graphviz
graph graphname { 
    a -- b; 
    b -- c;
    b -- d;
    d -- a;
}
```

<br>

```graphviz
digraph graphname{
    a -> b;
    b -> c;
    a -> c;
}
```

### 不同形状

```graphviz
digraph graph_name{
    //bgcolor="transparent";//背景透明
    
    subgraph cluster_subgraph_name{//聚集子图
        node[shape=box];
        cluster_A -> cluster_B;
    }
    
    subgraph subgraph_name{//子图
        node[shape=none];
        sub_A -> sub_B;
    }
    
    {//匿名子图
        node[shape=octagon];
        nest_A -> nest_B;
    }
    
    global_A -> global_B;
    
    cluster_B -> global_B;
    sub_B -> global_B;
    nest_B -> global_B;
}
```

### Hash MAP

```graphviz
digraph g {
      label = "Example of Hash Map"
      labelloc = "t";

	nodesep = .05;
	rankdir = LR;
	
	node[shape = record, width = .1, height = .1];
	
	node0[label = "<f0> |<f1> |<f2> |<f3> |<f4> |<f5> |<f6> |", height = 2.5];
	//我是一个节点，我有7个属性
	node [width = 1.5];
	node1[label = "{<n> n14 | 719 |<p>}"];
	//我还是一个节点， 也定义了三个属性
	node2[label = "{<n> a1 | 719 |<p>}"];
	node3[label = "{<n> i9 | 512 |<p>}"];
	node4[label = "{<n> e5 | 632 |<p>}"];
	node5[label = "{<n> t20 | 959 |<p>}"];
	node6[label = "{<n> o15 | 794 |<p>}"];
	node7[label = "{<n> s19 | 659 |<p>}"];

	//好了，我开始连接了
	node0:f0->node1:n;
	node0:f1->node2:n;
	node0:f2->node3:n;
	node0:f5->node4:n;
	node0:f6->node5:n;
	node2:p->node6:n;
	node4:p->node7:n;
}
```

### 复杂关系图

```graphviz
digraph yours_truly{
	splines=true;
	//newrank=true;
	nodesep="0.25";

	label = "Complex Architecture";
	labelloc = "t";

	subgraph cluster_clients_mobvm_a{
		label="xvm_a Clients";
		style=dashed;
		mobvm_a_req[label="xvm_a Application .." shape=Mrecord ];
		mobvm_a_req1[label="xvm_a Application 1" shape=Mrecord ];

		{rank=same; mobvm_a_req1; mobvm_a_req; } 
	}

	subgraph cluster_clients_prot_a{
		label="Z Clients";
		style=dashed;
		z_req[label="Z Application .." shape=Mrecord ];
		z_req1[label="Z Application 1" shape=Mrecord ];

		{rank=same; z_req1; z_req; }
	}

	subgraph cluster_prot_asrv{
		label="Z Shapeshifters\n(Configurable\nZ Shapeshifter)";
		style=dashed;

		z_shapeshifter[label="Z Shapeshifter Appearances" shape=Mrecord ];
		
		z_req1 -> z_shapeshifter [dir="both" constraint=true];
		z_req -> z_shapeshifter [dir="both" constraint=true];
	}

	subgraph cluster_machsrv {
		label="Machsrv Shapeshifter";
		machsrv_xprot_gen [label="<f0>Machsrv Core|<f1>PROT_A-to-XPROT Converter" shape=Mrecord ];
	
		z_shapeshifter -> machsrv_xprot_gen:f0 [label="Foo SCADA" dir="both" constraint=true];
		mobvm_a_req1 -> machsrv_xprot_gen:f0 [dir="both" constraint=true];
		mobvm_a_req -> machsrv_xprot_gen:f0 [dir="both" constraint=true];
	}

	subgraph cluster_accel {
		label="NOXd Shapeshifter";
		accel_appearance [label="NOXd Appearance" shape=Mrecord];
	}

	subgraph cluster_yours_trulybox {	
		label = "YoursTruly";

		subgraph cluster_tunnel{
			label = "Foo Reactor";
			tunnel_ingress [label="Ingress Shapeshifter" shape=Mrecord];
			accel [label="NOX Shapeshifter" shape=Mrecord];
			tunnel_egress [label="Egress Shapeshifter" shape=Mrecord];

			tunnel_ingress -> accel [dir="both"];
			accel -> tunnel_egress [dir="both"];

			accel_appearance -> accel [dir="both" constraint=true];
		}

		subgraph cluster_frontend{
			label = "YoursTruly Front-End";
			others[label="Other Chunks" shape=Mrecord];
			other_xprot [label="XPROT \"crystalizer\"" shape=Mrecord];
			front_end_jz[label="JZ Chunk" shape=Mrecord];	
			arbiter_vadam [label="Arbiter Chunks" shape=Mrecord];	

			arbiter_vadam -> other_xprot [style=dotted];
			arbiter_vadam -> front_end_jz [style=dotted];
			arbiter_vadam -> others [style=dotted];

			front_end_jz -> tunnel_ingress [dir="both" label="XPROT over PROT_G"];
			front_end_jz -> other_xprot[dir="both"];
			other_xprot -> machsrv_xprot_gen [dir="both" label="XPROT over PROT_G" constraint=true];
		}
	}

	subgraph cluster_shapeshifter_misc{
		label="Other Shapeshifters";
		style=dashed;
		handle_x[label="Shapeshift X" shape=Mrecord ];
		handle_1[label="Shapeshift 1" shape=Mrecord ];

		others -> handle_1 [style=dotted dir="both" constraint=true];
		others -> handle_x [style=dotted dir="both" constraint=true];
	}

    subgraph cluster_space_ship_a {
		label = "ShipX";
		
		subgraph cluster_vm_a {
			label="ReactorX";
			subgraph cluster_tungsten {
				label="Tungsten" 
				zzz [label="ZZZ" shape=Mrecord];
				yyy [label="YYY" shape=Mrecord];
			}
		}

		subgraph cluster_piston {
			label = "ReactorY";
        		zzz_worker [label="{ZZZ Worker}" shape=Mrecord ];
        		yyy_worker [label="{YYY Worker}" shape=Mrecord ];
        		foo_agent [label="{Foo Agent}" shape=Mrecord];
			
			foo_agent -> zzz_worker[dir="both" constraint=true label="MechCmd"];
			foo_agent -> yyy_worker[dir="both" constraint=true label="MechCmd"];
		}

		zzz_worker -> zzz [dir="both" constraint=true label="SCADA_A()"];
		yyy_worker -> yyy [dir="both" constraint=true label="SCADA_A()"];

	}

	tunnel_egress -> foo_agent [dir="both" label="Raw PROT_G" constraint=true];
}

```

## Footer on printed PDF

仅在导出 PDF 时显示

```javascript
<footer id="footer"></footer>

<script type="text/JavaScript">
var today = new Date();
var dd = String(today.getDate()).padStart(2, '0');
var mm = String(today.getMonth() + 1).padStart(2, '0'); //January is 0!
var yyyy = today.getFullYear();
today = yyyy + "-" + mm + "-" + dd;
document.getElementById("footer").innerHTML = today
</script>
```

### Footer and modified date

```javascript
## Revision

Revision | Author | Modification
| --- | --- | --- |
| JC | create the document

<footer id="footer"></footer>
<font color=#9c9c9c size=2><p id="lastmodifieddate"></p></font>

<script type="text/JavaScript">
var today = new Date();
var dd = String(today.getDate()).padStart(2, '0');
var mm = String(today.getMonth() + 1).padStart(2, '0'); //January is 0!
var yyyy = today.getFullYear();
today = yyyy + "-" + mm + "-" + dd;
document.getElementById("footer").innerHTML = today
today = new Date(document.lastModified);
document.getElementById("lastmodifieddate").innerHTML = "<br>更新日期：" + today
</script>
```

## Revision

Revision | Author | Comment
| --- | --- | --- |
190329 | JC | create the document

[回到首页](#top)

<footer id="footer"></footer>
<font color=#9c9c9c size=2><p id="lastmodifieddate"></p></font>

<script type="text/JavaScript">
var today = new Date();
var dd = String(today.getDate()).padStart(2, '0');
var mm = String(today.getMonth() + 1).padStart(2, '0'); //January is 0!
var yyyy = today.getFullYear();
today = yyyy + "-" + mm + "-" + dd;
document.getElementById("footer").innerHTML = today
today = new Date(document.lastModified);
document.getElementById("lastmodifieddate").innerHTML = "<br>更新日期：" + today
</script>

**Comment by legoming** [comment]
[000411B - Markdown Template Help.md](https://github.com/typora/typora-issues/files/8094299/000411B.-.Markdown.Template.Help.md)

**Comment by legoming** [comment]
> 000411B - Markdown Template Help.md

This is one of the file which cannot render in new Typora.

**Comment by abnerlee** [comment]
fixed in 1.1.3
